compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/selection/TextFieldSelectionManager.kt

```diff
@@ -461,10 +461,7 @@ internal class TextFieldSelectionManager(val undoManager: UndoManager? = null) {
                 updateFloatingToolbar(show = true)
             }
 
-            override fun onCancel() {
-                draggingHandle = null
```

Comment:
The code looks reasonable (The same happened in OnCancel).
Could you please elaborate, what fixes this removal?

Reply:
It does look reasonable indeed.
My point here is that we should upstream this to AOSP, and the AOSP doesn't have these lines and doesn't have the stuck magnifier bug. So I have tested that on physical iOS and Android devices, and I couldn't reproduce the issue without these lines.
Although I remember that bug too, I think we can remove these lines if we can't reproduce it now.

Reply:
Can we upstream these lines to be on the safe side?

Reply:
Or instead we can write some test in AOSP to make sure that we won't have regression here